### PR TITLE
Core and Community Packages

### PR DESCRIPTION
**Pull request by davidchase**

### CODE REVIEW

**Comment by TylorS** [comment]
LGTM

File: PACKAGES.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+### Most Core Packages & Examples
+| Name          | Description           
+| ------------- |:-------------:
+| [prelude](https://github.com/mostjs/prelude) | Base functional programming utils for mostjs packages
+| [stream](https://github.com/mostjs/stream)     | Core stream type
```
Comment (briancavalier) [inline PACKAGES.md]: Let's remove this one for now.  We're not using it yet, so I'd rather not promote it.

File: PACKAGES.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+### Most Core Packages & Examples
```
Comment (briancavalier) [inline PACKAGES.md]: Do you think we should try to (sub)group/categorize these by their purpose?  For example, `prelude` is a very different kind of thing than `hold`.  Maybe we should group "tools for most.js contributors" like `prelude` and `package-starter`, and "most.js stream add-ons" or some such?

Does that seem like it'd be helpful?

File: PACKAGES.md (hunk 1 above)
Comment (davidchase) [inline PACKAGES.md]: remove `stream` ?

**Comment by davidchase** [comment]
made some adjustments let me know what you guys think

File: PACKAGES.md (hunk 1 above)
Comment (briancavalier) [inline PACKAGES.md]: Yeah

**Comment by briancavalier** [comment]
lgtm!  Thanks @davidchase!

**Pull request merged**